.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.2
```

Comment:
I agree. In fact, we need to standardize our versions of Postgresql, `npm`, and `node.js` as well.